python/dgl/distributed/nn/pytorch/sparse_emb.py

```diff
@@ -77,8 +77,10 @@ def __init__(self, num_embeddings, embedding_dim, name=None,
         if th.distributed.is_initialized():
             self._rank = th.distributed.get_rank()
             self._world_size = th.distributed.get_world_size()
-        else:
-            assert 'th.distributed shoud be initialized'
+        # [TODO] The following code is clearly wrong but changing it to "raise DGLError"
+        # actually fails unit test.  ???
+        # else:
+        #     assert 'th.distributed should be initialized'
```

Comment:
I'm not going to handle this issue in this PR because this PR is about pylint only (and will affect many other PRs).  I'll leave it to future.

@VoVAllen for noting.

Reply:
cc @classicsong